contracts/validator-manager/interfaces/IPoSValidatorManager.sol

```diff
@@ -247,4 +290,8 @@ interface IPoSValidatorManager is IValidatorManager {
      * @param validationID The ID of the validation period being ended.
      */
     function claimDelegationFees(bytes32 validationID) external;
+
+    function changeValidatorRewardRecipient(bytes32 validationID, address recipient) external;
+
+    function changeDelegatorRewardRecipient(bytes32 delegationID, address recipient) external;
```

Comment:
Can you add natspec docs here? Should be pretty easy to copy the format from the other functions in this file